### PR TITLE
Add Zero Escape: Virtue's Last Reward

### PR DESCRIPTION
**Pull request by lonewolf329**
Adds Zero Escape: Virtue's Last Reward.

*Automatically generated by PR builder*

### CODE REVIEW

File: zero-escape-virtues-last-reward/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Zero Escape : Virtue's Last Reward"
```
Comment (j-selby) [inline zero-escape-virtues-last-reward/game.dat]: Is a space needed here in the title?

File: zero-escape-virtues-last-reward/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+description = "Zero Escape: Virtue's Last Reward[a] is an adventure video game developed by Chunsoft. The story follows the player character Sigma, a man who is abducted and forced along with eight other individuals to play the Nonary Game, which puts its participants in a life-or-death situation. As the story progresses, the characters begin to unravel the secrets behind the Nonary Game, as well as its true purpose."
```
Comment (j-selby) [inline zero-escape-virtues-last-reward/game.dat]: `[a]` - if you are going to copy from Wikipedia, at least correct the formatting 😛

File: zero-escape-virtues-last-reward/game.dat [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "0004000000096700"
+region = "usa"
+release_date = "2012-02-16"
```
Comment (TheKoopaKingdom) [inline zero-escape-virtues-last-reward/game.dat]: There needs to be a testcase here.

File: zero-escape-virtues-last-reward/game.dat (hunk 2 above)
Comment (lonewolf329) [inline zero-escape-virtues-last-reward/game.dat]: damn you caught me 😄

File: zero-escape-virtues-last-reward/game.dat (hunk 3 above)
Comment (lonewolf329) [inline zero-escape-virtues-last-reward/game.dat]: done

File: zero-escape-virtues-last-reward/game.dat (hunk 1 above)
Comment (lonewolf329) [inline zero-escape-virtues-last-reward/game.dat]: Thanks for mentioning that. Well it was needed so i removed it now.

File: zero-escape-virtues-last-reward/game.dat [hunk 4]
```diff
@@ -0,0 +1,19 @@
+title = "Zero Escape: Virtue's Last Reward"
+description = "Zero Escape: Virtue's Last Reward is an adventure video game developed by Chunsoft. The story follows the player character Sigma, a man who is abducted and forced along with eight other individuals to play the Nonary Game, which puts its participants in a life-or-death situation."
+needs_system_files = false
```
Comment (TheKoopaKingdom) [inline zero-escape-virtues-last-reward/game.dat]: If the game has major graphical or audio glitches, you should add the relevant GitHub issues.

File: zero-escape-virtues-last-reward/game.dat [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+[[releases]]
+title = "0004000000096700"
+region = "usa"
+release_date = "2012-02-16"
+
+[[ testcases ]]
+title = "0004000000096700"
+compatibility = "2"
+date = "2017-07-08"
+version = "	Nightly Build - 2e37ce0"
```
Comment (TheKoopaKingdom) [inline zero-escape-virtues-last-reward/game.dat]: The extra spaces here are unnecessary.

File: zero-escape-virtues-last-reward/game.dat [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+author = "lonewolf329"
+cpu = "Intel Core 2 duo cpu e7500"
```
Comment (TheKoopaKingdom) [inline zero-escape-virtues-last-reward/game.dat]: `Duo` and `CPU` here should have proper capitalization.

File: zero-escape-virtues-last-reward/game.dat [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+gpu = "GeForce GT 640"
+os = "Windows 7 ultimate sp1"
```
Comment (TheKoopaKingdom) [inline zero-escape-virtues-last-reward/game.dat]: `Ultimate` and `SP1` here should have proper capitalization.

File: zero-escape-virtues-last-reward/game.dat [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+[[releases]]
+title = "0004000000096700"
+region = "usa"
+release_date = "2012-02-16"
+
+[[ testcases ]]
+title = "0004000000096700"
+compatibility = "1"
+date = "2017-07-08"
+version = "Nightly Build-2e37ce0"
```
Comment (j-selby) [inline zero-escape-virtues-last-reward/game.dat]: This isn't a valid Git commit. This should be `HEAD-2e37ce0`

Comment (lonewolf329) [inline zero-escape-virtues-last-reward/game.dat]: Corrected.

**Pull request merged**